Lib/test/test_typing.py

```diff
@@ -4315,7 +4316,7 @@ def test_basics_keywords_syntax(self):
         self.assertEqual(Emp.__total__, True)
 
     def test_typeddict_special_keyword_names(self):
-        TD = TypedDict("TD", cls=type, self=object, typename=str, _typename=int, fields=list, _fields=dict)
+        TD = TypedDict("TD", {'cls': type, 'self': object, 'typename': str, '_typename': int, 'fields': list, '_fields': dict})
```

Comment:
Ah, there's an issue with this line. It looks like this test is specifically testing that the behaviour for the kwargs syntax is the same as with the literal-dict syntax. (The same behaviour is tested for the literal-dict syntax in `test_basics_functional_syntax`). So I don't think the change to this line makes sense.

Maybe just keep this `TD` definition as it is, but put it in another `self.assertWarns(DeprecationWarning)` block?

Reply:
Done!

Reply:
Thanks!